examples/image_example.py

```diff
@@ -0,0 +1,71 @@
+"""
+Example that enables images from a camera and writes a single frame to a file.
+"""
+
+import argparse
+import asyncio
+import logging
+import os
+import qtm_rt
+import xml.etree.ElementTree as ET
+
+
+def output_format(output_path):
+    extension = os.path.splitext(output_path)[1].lower()
+    if extension == ".jpg":
+        return "jpg"
+    if extension == ".png":
+        return "png"
+
+    raise RuntimeError("Unsupported output format {extension}")
+
+
+def enable_disable_cameras(output_path, target_camera_id, xml_str):
+    format = output_format(output_path)
+    xml = ET.fromstring(xml_str)
+    for camera in xml.findall("./Image/Camera"):
+        camera_id = camera.find("ID").text
+        is_target_camera = camera_id == target_camera_id
+        logging.info("Setting Enabled for Camera %s to %s", camera_id, is_target_camera)
+        camera.find("Enabled").text = str(is_target_camera)
+
+        if is_target_camera:
+            camera.find("Format").text = format
+
+    xml.tag = "QTM_Settings"
+    return ET.tostring(xml).decode("utf-8")
+
+
+async def main(password, target_camera_id, output_path):
+    connection = await qtm_rt.connect("127.0.0.1")
+
+    if connection is None:
+        raise RuntimeError("Failed to connect")
```

Comment:
Is it helpful at all to provide the **target_camera_id** or **output_path** here in the RunetimeError message? If the likelihood of it being helpful is 0% feel free to just ignore this comment, else consider adding one or the other, or both.

Reply:
Good idea in general to include more info in error messages. Though, in this case, the failure to connect have nothing to do with the `target_camera_id` or the `output_path`. So including them is not useful.